### ISSUE TITLE
Spinner stops working after start using of AndroidAnnotations

### ISSUE THREAD

**Issue by SammoLove** [androidannotations/androidannotations#1883]
After adding @EActivity all my Spinners become empty.
The main problem in logcat is Could not find class 'android.widget.ThemedSpinnerAdapter'
Full fragment of log:
[fr.txt](https://github.com/androidannotations/androidannotations/files/553156/fr.txt)

(minSdkVersion 17, targetSdkVersion 25, compileSdkVersion 25)

**Comment by WonderCsabo** [comment]
It is hardly related to AndroidAnnotations. It does not have effect on the Android runtime on your device.

**Comment by SammoLove** [comment]
If I remove @EActivity and '_' in manifest, my dropdown lists continues to operate normally. You can check this fact by yourseft by using this code:
```
final String[] dateFormat = {"31.12.2016", "31/12/2016", "12-31-2016", "December 31, 2016"};

		ArrayAdapter adapter = new ArrayAdapter<>(this, android.R.layout.simple_spinner_dropdown_item, dateFormat);
		adapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);

		Spinner spinnerDateFormat = (Spinner) findViewById(R.id.spinnerDateFormat); //or by @ViewById
		spinnerDateFormat.setAdapter(adapter);
```

**Comment by WonderCsabo** [comment]
Can you share the project?

**Comment by SammoLove** [comment]
I created a mini-project for you, and when created it, I realized what the problem was.
I filled the lists in the method onCreate ().
I used to get View via findById and worked with them.
Now, as it turned out during debugging, all in View in onCreate is not created!
The problem was solved when I found an annotation @AfterViews in the docs, and the method under this annotation now fills all my actions and does initialization of fields.
It would be good to add in Get started> Start using the documentation, as the basic instructions.
